### DISCUSSION TITLE
WebView2 Installer Issues

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [wailsapp/wails#2220] [Q&A]
We created a Wails app using the embedded WV2 installer. Now, we have one user/case where the installer kicks in, but things don't work.

1. After starting the program, the UAC dialog asks to allow "Microsoft Edge Update" to make changes to the system:
![68747470733a2f2f692e696d6775722e636f6d2f744e546f3831562e706e67](https://user-images.githubusercontent.com/1826391/208457643-85be934d-6517-48c6-af81-3c666de5bf8a.png)

2. After confirming the WebView2 installer pops up again, and fails. Now with error message: no internet connection, HTTP 403:
![68747470733a2f2f692e696d6775722e636f6d2f6e425a694638582e706e67](https://user-images.githubusercontent.com/1826391/208457822-a648ea4b-a603-47b8-849e-3e9ba747deb1.png)
The error message states that the proxy configuration should be checked.

Since the Wv2 installer is a MS component I would expect it to work using the current proxy settings.

Is this a known problem, does anyone has an idea what to check or do?

**Comment by stffabi** [comment]
Maybe the embedded bootstrapper needs an update. Have you tried what happens wehen downloading the latest bootstrapper from microsoft and start that one?
